src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -194,9 +194,10 @@ default StudentParticipation findByExerciseIdAndStudentIdAndTestRunWithEagerSubm
             LEFT JOIN FETCH r.submission s
             WHERE p.exercise.id = :#{#exerciseId}
                 AND (r.id = (SELECT max(id) FROM p.results)
+                    OR r.assessmentType <> 'AUTOMATIC'
                     OR r IS NULL)
             """)
-    Set<StudentParticipation> findByExerciseIdWithLatestResult(@Param("exerciseId") Long exerciseId);
+    Set<StudentParticipation> findByExerciseIdWithLatestResults(@Param("exerciseId") Long exerciseId);
```

Comment:
imo you should include `manual` in the method name. As you have a manual query, that's not an issue anyway.